### ISSUE TITLE
Query builder throwing error when chaining aggregates

### ISSUE THREAD

**Issue by circulon** [MasoniteFramework/orm#398]
**Describe the bug**
I couldnt find a word to describe a group category for query commands like COUNT, AVG, MAX, MIN etc so please adjust the title as required 

When using the query builder and multiple chained meta commands like .count, .max, .avg an invalid query is built.

**To Reproduce**
example
```        
    builder = (
            QueryBuilder().from_('table_a')
            .join('table_b', 'table_b.id', '=', 'table_a.table_b_id')
            .count('table_a.id as table_a_count')
            .avg('table_b.rate as table_b_avg_rate')
            .max('table_b.rate as table_b_max_rate')
            .where('table_a.user_id', '=', '30745508-bc6f-49fd-a635-18f5d96f5afe')
    )
    print(builder.tosql())
```
Generates
```
SELECT COUNT("table_a"."id") AS table_a_countAVG("table_b"."rate") AS table_b_avg_rateMAX("table_b"."rate") AS table_b_max_rate FROM "table_a" INNER JOIN "table_b" ON "table_b"."id" = "table_a"."table_b_id" WHERE "table_a"."user_id" = '30745508-bc6f-49fd-a635-18f5d96f5afe' 
```

In this context table_a is a link table between a user table and other related data in table_b 

**Expected behavior**
Expected valid generated query like this
 
```
SELECT COUNT("table_a"."id") AS table_a_count, AVG("table_b"."rate") AS table_b_avg_rate, MAX("table_b"."rate") AS table_b_max_rate FROM "table_a" INNER JOIN "table_b" ON "table_b"."id" = "table_a"."table_b_id" WHERE "table_a"."user_id" = '30745508-bc6f-49fd-a635-18f5d96f5afe' 
```

**Screenshots or code snippets**
Screenshots help a lot. If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Mac OSX 
 - Version Cata;ina 10.15.7

**What database are you using?**
 - Type: [e.g. Postgres, 
 - Version 10.7
 - Masonite ORM v1.0.31

**Comment by josephmancuso** [comment]
They are called "aggregates" lol. I'll work on this

**Comment by josephmancuso** [comment]
Fixed in version 1.0.32
